vispy/testing/_runners.py

```diff
@@ -190,8 +190,14 @@ def _check_line_endings():
 """
 
 
-def _examples():
-    """Run all examples and make sure they work
+def _examples(individial_file_paths):
+    """Run examples and make sure they work.
+
+    Parameters
+    ----------
+
+    individial_file_paths: str
```

Comment:
I'm also not a fan of re-using variables. The two are conceptually different - why reuse the same variable name?